index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Haliljon</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container"
+        ><a href="#" class="link">Welcome</a>
+
+        <a href="#" class="menu-btn"
+          ><ion-icon name="menu-outline"></ion-icon></a></div
+    ></header>
+    <main>
+      <section class="section-home">
+        <div class="primary"
+          ><h1 class="heading-primary">Hey there. I’m Halijon </h1>
+          <h1 class="heading-primary2">I’m a software developer</h1></div
+        >
+        <div class="supporting">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p></div
```

Comment:
This is almost the same, but you can make it completely the same as Figma if you set the `line-height: 28px;`